### PR TITLE
feat!: move to API to python microgenerator

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,7 +24,9 @@
 version = "0.4.0"
 release_status = "Development Status :: 4 - Beta"
 dependencies = [
-    "google-api-core[grpc] >= 1.14.0, < 2.0.0dev",
+    "google-api-core[grpc] >= 1.21.0, < 2.0.0dev",
+    "libcst >= 0.2.5",
+    "proto-plus >= 1.1.0",
     'enum34; python_version < "3.4"',
```
Comment (busunkim96) [inline setup.py:30]: Please remove the `enum` dependency (since the library is only compatible with 3.6+)

Comment (xiaozhenliu-gg5) [reply setup.py]: 👍 done and noted!

File: UPGRADING.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+uri = ''
+response = client.report_phishing(parent, uri)
+```
+
+
+**After:**
+```py
+from google.cloud import phishingprotection_v1beta1
+client = phishingprotection_v1beta1.PhishingProtectionServiceClient()
+response = client.search_hashes(request={"parent": "project/project_number", "uri": "''"})
```
Comment (software-dov) [inline UPGRADING.md:46]: I think it would be more helpful for the before and after examples to use the same method.

Comment (xiaozhenliu-gg5) [reply UPGRADING.md]: Thanks! it's updated to `report_phishing`

**Pull request merged**